auth/auth/auth.py

```diff
@@ -789,7 +789,7 @@ def run():
     web.run_app(
         deploy_config.prefix_application(app, 'auth'),
         host='0.0.0.0',
-        port=5000,
+        port=443,
```

Comment:
I'm not being clear enough. I meant do you need to change this to 8443?

Reply:
No, 8443 is the port that gateway is listening on. It then forwards requests to auth:443